fluent-package/msi/assets/fluentd.bat

```diff
@@ -12,6 +12,8 @@ set FLUENT_PACKAGE_TOPDIR=%FLUENT_PACKAGE_TOPDIR:\=/%
 
 set PATH=%FLUENT_PACKAGE_TOPDIR%bin;%PATH%
 set PATH=%FLUENT_PACKAGE_TOPDIR%;%PATH%
+set "GEM_HOME=%FLUENT_PACKAGE_TOPDIR%lib\ruby\gems\%RUBY_VERSION%"

```

Comment:
where %RUBY_VERSION% definition come from?

Reply:
Ah, I missed the code to set up `%RUBY_VERSION%`.
Thanks

Reply:
Fixed with https://github.com/fluent/fluent-package-builder/pull/722/commits/1922b2909326289c461b5e96b2ee4a3b0c60488c